microsoft-outlook/latest/install/index.md

```diff
@@ -89,7 +89,7 @@ transform-outlook
 
 > **Note:** Alfresco customers can request Quay.io credentials by logging a ticket at [Alfresco Support](https://support.alfresco.com/){:target="_blank"}. These credentials are required to pull private (Enterprise-only) Docker images from Quay.io.
 
-> **Note:** Make sure that you request credentials for Alfresco Content Services and Alfresco Outlook Integration, so that you can use the additional `transform-outlook-1.0.x` Docker image. Version 1.0.0 of the Outlook T-Engine can also be used for the latest Outlook Integration release (version 2.8.1).
+> **Note:** Make sure that you request credentials for Alfresco Content Services and Alfresco Outlook Integration, so that you can use the additional `transform-outlook-1.1.x` Docker image. Version 1.1.0 of the Outlook T-Engine can also be used for the previous Outlook Integration releases (version 2.8.x).
```

Comment:
Hi @mhofmann-itnovum, I had a re-think about what we discussed earlier today about this last sentence - see https://github.com/Alfresco/docs-alfresco/pull/553#pullrequestreview-904354678. It was added when we released Outlook Integration 2.8.1 to indicate to customers that there wasn't an updated Outlook T-Engine image between 2.8 & 2.8.1. Happy to take your advice on how we should phrase this.

Reply:
Hi @anxumalo , I think it should be clear the way you phrased it. 👍